optimizely/decision_service.py

```diff
@@ -12,6 +12,7 @@
 # limitations under the License.
 
 from collections import namedtuple
+from six import string_types
```

Comment:
This is a third party library kindly make sure that its mentioned in the requirements.

Reply:
why we need to use it?

Reply:
Looks like it's already being used.